### PR TITLE
9.0 postlogisitcs config helper

### PR DESCRIPTION
**Pull request by yvaucher**
Depends on 
- [x] https://github.com/OCA/carrier-delivery/pull/73
- [x] https://github.com/OCA/server-tools/pull/411

### CODE REVIEW

File: delivery_carrier_label_postlogistics/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+# © 2013-2015 Yannick Vaucher (Camptocamp SA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (bealdav) [inline delivery_carrier_label_postlogistics/models/__init__.py]: Lines above not useful, there is no real code in this file, and it's in ascii, the defaut coding, save the cloud planet of unsueful bytes ;-)

File: delivery_carrier_label_postlogistics/postlogistics/web_service.py [hunk 2]
```diff
@@ -78,8 +60,7 @@ def _send_request(self, request, **kwargs):
         except Exception as e:
             # if authentication error
             if isinstance(e[0], tuple) and e[0][0] == 401:
-                raise orm.except_orm(
-                    _('Error 401'),
+                raise exceptions.Warning(
```
Comment (bealdav) [inline delivery_carrier_label_postlogistics/postlogistics/web_service.py]: s/Warning/UserError

**Comment by bealdav** [comment]
LGTM, no test

File: delivery_carrier_label_postlogistics/models/__init__.py (hunk 1 above)
Comment (yvaucher) [inline delivery_carrier_label_postlogistics/models/__init__.py]: Ok this is out of scope of this PR, please review only the last commit.

File: delivery_carrier_label_postlogistics/__openerp__.py [hunk 3]
```diff
@@ -1,40 +1,25 @@
...
-          'delivery_view.xml',
-          'res_config_view.xml',
+ 'data': ['data/res_partner.xml',
+          'data/delivery.xml',
+          'views/delivery.xml',
+          'views/postlogistics_license.xml',
+          'views/res_config.xml',
           'security/ir.model.access.csv',
           ],
  'tests': [],
```
Comment (tafaRU) [inline delivery_carrier_label_postlogistics/__openerp__.py]: Please remove it

**Comment by tafaRU** [comment]
Also please fix https://travis-ci.org/OCA/carrier-delivery/jobs/121696752#L464

**Comment by yvaucher** [comment]
Could we validate #73 first ?

**Comment by yvaucher** [comment]
All comments fixed and commited

**Comment by yvaucher** [comment]
Rebased
